validation/standalone-constraint-constraint.ttl

```diff
@@ -26,8 +26,8 @@
     ] ;
     sh:property [
         sh:path sh:property ;
-        sh:node <ObservationConstraintProperty>;
-        sh:message "The constraints do not validate"
+        sh:node <ObservationConstraintProperty>, <PropertyWithName>, <PropertyWithType> ;
+        sh:message "The constraints do not validate";
     ] ;
```

Comment:
Now that you're touching this, what do you think about splitting this entire property into multiple, each with its own `sh:node` and dedicated message. This way we can have better feedback that a generic "The constraints do not validate"


```suggestion
        sh:node <ObservationConstraintProperty> ;
        sh:message "The constraints do not validate";
    ] ,
    [
        sh:path sh:property ;
        sh:node <PropertyWithName> ;
        sh:message "Every dimension needs a schema:name";
    ] ,
    [
        sh:path sh:property ;
        sh:node <PropertyWithType> ;
        sh:message "Every dimension needs a sh:datatype, sh:nodeKind or sh:datatype within sh:or (...)";
    ];
```

I think it is also worth doing for all nested properties of `<ObservationConstraintProperty>`

Reply:
The other property shapes in `<ObservationConstraintProperty>` already provide their specific messages in result details. See for example https://s.zazuko.com/21HjLvb.
`<PropertyWithName>` and `<PropertyWithType>` need separate node shapes because they have `sh:or` which discards the details

Reply:
Ok, splitting the shapes is worth it nevertheless. This also makes them no more anonymous so they can be referenced (and deactivated) in importing modules (looking at you `<PropertyInList>` )